quickstarts/deploy-vm/main.go

```diff
@@ -0,0 +1,159 @@
+package main
+
+import (
+	"context"
+	"encoding/json"
+	"io/ioutil"
+	"log"
+
+	"github.com/Azure/azure-sdk-for-go/profiles/2017-03-09/network/mgmt/network"
```

Comment:
if we use a profile here it should be "latest"

Reply:
Locking to a specific version or profile is indeed what we'd recommend, you're right :). Since we only have one profile at the moment though and it's out of date with Azure APIs, for now you should snap to a specific version rather than a profile, e.g. `github.com/Azure/azure-sdk-for-go/services/network/mgmt/2017-09-01/network`.

In the future we'll have more usable profiles for regular Azure too, stay tuned :)